ast/builtins.go

```diff
@@ -284,17 +284,6 @@ var DefaultBuiltins = [...]*Builtin{
 // built-in definitions.
 var BuiltinMap map[string]*Builtin
 
-// IgnoreDuringPartialEval is a set of built-in functions that should not be
-// evaluated during partial evaluation. These functions are not partially
-// evaluated because they are not pure.
-var IgnoreDuringPartialEval = []*Builtin{
```

Comment:
⚠️ Removing a public field is breaking compatibility. We'll need to call this out, and mention a workaround in the release notes.